milestones/data.py

```diff
@@ -74,37 +74,37 @@ def _activate_milestone(milestone, propagate=True):
     """
     Activates an inactivated (soft-deleted) milestone as well as any inactive relationships
     """
-    # We use models.Value(0) to make use of the indexing on the field. MySQL does not
+    # We use models.Value(False) to make use of the indexing on the field. MySQL does not
     # support boolean types natively, and checking for False will cause a table scan.
```

Comment:
@qasimgulzar -- I'm not sure this comment makes much sense anymore with the change. Also, based on the original comment, could this change possibly cause performance issues now for MySQL? Or is this no longer the case?